ArjunaJTS/idl/idlj-openjdk/pom.xml

```diff
@@ -81,6 +81,7 @@
             <configuration>
               <sources>
                 <source>${project.build.directory}/generated-sources/idl</source>
+                <source>${project.basedir}/../src/main/</source>
```

Comment:
@jmfinelli is this definitely needed I am not spotting a reference to this path in the diff?

Reply:
Ah, it's https://github.com/jbosstm/narayana/pull/2049/commits/dde6621d981b608faa6d4b2f307919dae134a869#diff-d4f42289a362c7d991ef7305ebf659ea59d80c9fcb0e3ee42fc80f79f457aa92R59

Reply:
Yes, it is. Basically, this line is packing `idl` files in the source jar